xbmc/video/VideoDatabase.cpp

```diff
@@ -2687,13 +2687,21 @@ void CVideoDatabase::GetBookMarksForFile(const std::string& strFilenameAndPath,
         bookmark.playerState = m_pDS->fv("playerState").get_asString();
         bookmark.player = m_pDS->fv("player").get_asString();
         bookmark.type = type;
-        if (type == CBookmark::EPISODE)
+        std::string strSQL2;
+        switch (type)
         {
-          std::string strSQL2=PrepareSQL("select c%02d, c%02d from episode where c%02d=%i order by c%02d, c%02d", VIDEODB_ID_EPISODE_EPISODE, VIDEODB_ID_EPISODE_SEASON, VIDEODB_ID_EPISODE_BOOKMARK, m_pDS->fv("idBookmark").get_asInt(), VIDEODB_ID_EPISODE_SORTSEASON, VIDEODB_ID_EPISODE_SORTEPISODE);
+        case CBookmark::EPISODE:
+          strSQL2 = PrepareSQL("SELECT c%02d, c%02d FROM episode WHERE c%02d=%i ORDER BY c%02d, c%02d", VIDEODB_ID_EPISODE_EPISODE, VIDEODB_ID_EPISODE_SEASON, VIDEODB_ID_EPISODE_BOOKMARK, m_pDS->fv("idBookmark").get_asInt(), VIDEODB_ID_EPISODE_SORTSEASON, VIDEODB_ID_EPISODE_SORTEPISODE);
           m_pDS2->query(strSQL2.c_str());
-          bookmark.episodeNumber = m_pDS2->fv(0).get_asInt();
-          bookmark.seasonNumber = m_pDS2->fv(1).get_asInt();
+          bookmark.title = StringUtils::Format("%s %li %s %li", g_localizeStrings.Get(20373).c_str(), m_pDS2->fv(1).get_asInt(), g_localizeStrings.Get(20359).c_str(), m_pDS2->fv(0).get_asInt());
```

Comment:
This whole string needs to be localizable and not put together in the code (think of RTL languages).